cylc/uiserver/handlers.py

```diff
@@ -386,6 +390,18 @@ def open(self, *args, **kwargs):  # noqa: A003
         )
 
     async def on_message(self, message):
+        with suppress(ValueError, SyntaxError):
+            message_dict = ast.literal_eval(message)
+            op_name = ''
+            op_id = message_dict.get("id", None)
+            with suppress(KeyError):
+                op_name = message_dict['payload']['operationName']
+            if (message_dict['type'] == 'start'
+                and op_name
+                    and message_dict['payload']['operationName'] == 'LogData'):
+                self.sub_statuses[op_id] = 'start'
+            if message_dict['type'] == 'stop' and op_id in self.sub_statuses:
+                self.sub_statuses[op_id] = 'stop'
```

Comment:
Because we only add entries to this dictionary (and never remove them) this is a memory leak albeit a very small one.

Is there an opportunity later on in the code, perhaps when the operation is "unsubscribed" later on in the process, to remove this entry from the `dict` to avoid them accumulating?